capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -31,13 +31,21 @@ function getClassrooms() {
     });
 }
 
+// TODO: Replace dummy values with real values in future PR
 function createClassroomDivElement(classroom) {
-    let domparser = new DOMParser();
-    let doc = domparser.parseFromString(`
-            <div class="classroom">
-                <h2>${classroom.entity.propertyMap.subject}</h2>
-                <p><a href="form.html">Classroom Link</a></p>
+    const domparser = new DOMParser();
+    const doc = domparser.parseFromString(`
+            <div class="card margin margin-left">
+                <img class="card-img-top" src="/assets/soundwave.svg" alt="Room Card">
+                <div class="card-body text-center">
+                    <h5 class="card-title">${classroom.entity.propertyMap.subject}</h5>
+                    <a class="card-text small-text" href="#">Kyle Smith</a>
+                    <div class="card-text small-text">Followers: Infinite</div>
+                    <div class="card-text small-text">Tag(s): </div>
+                    <div class="small-spacing-bottom"></div>
+                    <button type="button" class="btn btn-default" onclick="window.location.href='form.html'">View</button>
```

Comment:
I believe, but im not 100% this actually works, that you can just make this a <button ...><a href=>...</a></button>, and that would be more accessible since it would help for example screen readers realize that this is a link to somewhere and call that out

Reply:
Got it